deploy/servctl_utils/deploy_command_utils.py

```diff
@@ -75,7 +75,7 @@
     'nginx': ['{deploy_to}/tls.key', '{deploy_to}/tls.crt'],
     'seqr': [
         'omim_key', 'postmark_server_token', 'slack_token', 'airtable_key', 'django_key', 'seqr_es_password',
-        'google_client_id',  'google_client_secret'
+        '{deploy_to}/google_client_id',  '{deploy_to}/google_client_secret'
```

Comment:
Does the key path in `deploy/kubernetes/seqr/seqr.gcloud.yaml` need to be updated accordinarily?

Reply:
nope! This isn't changing where the yaml file is supposed to be, its only changing the secrets, which aren't in source control. If you restore the latest serets backup, yo'll see that I added nested folders for these, with different keysin the dev and prod folders